n3fit/src/n3fit/hyper_optimization/rewards.py

```diff
@@ -192,10 +192,14 @@ def compute_loss(
         pdf_model: MetaModel,
         experimental_data: List[DataGroupSpec],
         fold_idx: int = 0,
+        include_penalties=False,
```

Comment:
Hi @scarlehoff. Thank you for your amazing work in making this PR up. Would it be an option to `include_penalties=True` via runcard? I am asking this before pulling these changes to my snellius branch as the default `False` option would change the results I currently have.

Reply:
sure!
Since I'm at it, I'll also try to add the proportion of best replicas to the runcard so that it can be easily controlled.

Reply:
Added as `kfold:penalties_in_loss: True`

It defaults to `False`.

Please let me know if any of the changes I've done produce some effect somewhere else that I might have overlooked!